po/POTFILES

```diff
@@ -1,8 +0,0 @@
-share/lutris/ui/about-dialog.ui
-share/lutris/ui/dialog-lutris-login.ui
-share/lutris/ui/dialog-pga-sources.ui
-share/lutris/ui/dialog-uninstall-game.ui
-share/lutris/ui/runner-entry.ui
-share/lutris/ui/runner-remove-all-versions-dialog.ui
-share/lutris/ui/runner-remove-confirm-dialog.ui
-share/lutris/ui/runners-dialog.ui
```

Comment:
Please keep POTFILES. `autogen.sh` is more than sketchy, not every `.py` file is translatable. Also, one might want to add i18n support for the metainfo or .desktop file.

Reply:
Have added `POTFILES` back and deleted `autogen.sh`.